src/plugins/WebhookManager/index.tsx

```diff
@@ -0,0 +1,162 @@
+/*
+ * Vencord, a Discord client mod
+ * Copyright (c) 2024 Vendicated and contributors
+ * SPDX-License-Identifier: GPL-3.0-or-later
+ */
+
+import { ApplicationCommandInputType, ApplicationCommandOptionType, findOption, sendBotMessage } from "@api/Commands";
+import { Devs } from "@utils/constants";
+import { Logger } from "@utils/Logger";
+import definePlugin, { PluginNative } from "@utils/types";
+
+const Native = VencordNative.pluginHelpers.WebhookManager as PluginNative<typeof import("./native")>;
+const WMLogger = new Logger("WebhookManager");
+
+export default definePlugin({
+    name: "WebhookManager",
+    description: "Manage your webhooks easily; delete, send messages, get detailed info and more.",
+    authors: [Devs.Byron],
+    dependencies: ["CommandsAPI"],
+    commands: [
+        {
+            name: "webhook delete",
+            description: "Delete a webhook.",
+            inputType: ApplicationCommandInputType.BUILT_IN,
+            options: [
+                {
+                    name: "url",
+                    description: "The URL of the webhook",
+                    type: ApplicationCommandOptionType.STRING,
+                    required: true
+                }
+            ],
+            execute: async (option, ctx) => {
+                try {
+                    await fetch("" + findOption(option, "url"), { method: "DELETE" }).then(() => sendBotMessage(ctx.channel.id, { content: "The webhook has deleted successfully." }));
```

Comment:
try to avoid long lines! and prefer await over .then() unless there's a good reason to use the latter
```suggestion
                    await fetch(findOption(option, "url"), {
                    	method: "DELETE"
                  	});
                    sendBotMessage(ctx.channel.id, {
                    	content: "The webhook has deleted successfully."
                  	});
```